clients/client-python/gravitino/dto/responses/base_response.py

```diff
@@ -35,7 +35,6 @@ def code(self) -> int:
 
     def validate(self):
         """Validates the response code.
-        TODO: @throws IllegalArgumentException if code value is negative.
+        @throws IllegalArgumentException if code value is negative.
         """
-        if self._code < 0:
-            raise ValueError("code must be >= 0")
+        assert self._code >= 0, "code must be >= 0"
```

Comment:
Will it throw an `IllegalArgumentException` here?

Reply:
No, because `assert` can only throw `AssertError`, it can't throw custom exceptions. I think I will create another issue to fix all `assert` problem, otherwise this PR will include too many changes.

Reply:
Can you please create an issue to track this in case you forget?

Reply:
Have created issue #4100